### ISSUE TITLE
3D scene interactions not possible after opening dialogues on Big Sur

### ISSUE THREAD

**Issue by marine-3d** [Ultimaker/Cura#8745]
Cura 4.8
Mac OS 11.0.1
2 x UM3's

With a model on the build plate, if you click on the monitor tab and then back to prepare/preview, model no longer selectable with mouse.

No difference if model just loaded and/or not sliced/sent to printer. Can be selected with keyboard shortcut/right click - select all models, but no interaction is possible.

**Comment by nallath** [comment]
It seems that the issue template has not been filled in entirely. Please check our guide on https://github.com/Ultimaker/Cura/wiki/Reporting-issues for more information on what information we need and why we need it.

**Comment by marine-3d** [comment]
Hi Nallath,

Hopefully the attached video is enough to go on. Also, managed to isolate the issue to when the UM3 status is cloud or network connected (Blue icon with cloud or tick symbol). When the printers are off (grey icon), the issue isn't present and you can still select the model as normal, even when flicking between prepare/preview/monitor.

Hope this helps.

[Screen Recording 2020-11-13 at 16.48.21-1.mov.zip](https://github.com/Ultimaker/Cura/files/5538547/Screen.Recording.2020-11-13.at.16.48.21-1.mov.zip)

**Comment by fieldOfView** [comment]
Does this also happen without the sidebargui plugin installed?

**Comment by marine-3d** [comment]
Hi fieldOfView,

Just uninstalled the plugin but still the same issue.

**Comment by fieldOfView** [comment]
~Wew, so it isnt my fault for a change.~ Err, I meant... shame that does not fix the issue.

**Comment by scottmloving** [comment]
Same issue on my Macbook Pro with Ender 3. I just upgraded to Big Sur (11.0.1) and Cura 4.8.0. After monitoring a USB print, models are not selectable, even after starting a new project or reloading the models.When a model is selected from the object list, the move arrows appear, but they disappear as soon as they are clicked.

**Comment by Ghostkeeper** [comment]
Hmm, I'm unable to reproduce the issue. I'm basically doing the exact same steps as marine-3d, just with a different model and my own UM3.

Do either of you have a log file you can share with us?

In my log I'm only seeing this warning when switching to the monitor page:

    plugins/UM3NetworkPrinting/resources/qml/MonitorStage.qml:75:13: Unable to assign [undefined] to bool

Seems that the `supportPrintJobsQueue` property is not defined for LAN printers. But that shouldn't really affect anything.

**Comment by khaoskosmos** [comment]
I have this issue as well. MacOS Big Sur 11.0.1. If I go to the monitor tab and back, I can no longer select the model. Starting a new project does no clear the problem, only completely closing Cura (CMD + Q) and relaunching it.
[cura-no-select-macos.mp4.zip](https://github.com/Ultimaker/Cura/files/5578953/cura-no-select-macos.mp4.zip)

**Comment by nikolafamit** [comment]
Same problem. Big Sur & Cura 4.8

**Comment by marine-3d** [comment]
Log attached

[cura.log](https://github.com/Ultimaker/Cura/files/5590281/cura.log)

**Comment by evtrados** [comment]
Hey all.
Just upgraded to Big Sur v11.0.1 from Big Sur beta, but I cannot reproduce the issue. I used .stl and .3mf files and 3 different models for each. I also tried removing the configuration folder, but still the issue is not there. Could it be model-specific? I doubt it, but in any case if you could send me the model with which you can reproduce the issue, or even better the whole configuration folder, I can try and check if it's appearing then. 
Thanks!

**Comment by khaoskosmos** [comment]
It doesn't always do it, I'm not sure what triggers it. I am not signed in to Cura at all.

I just had it happen after leaving Cura open after a print. I imported a new stl and could not select it (you can get it to select by click on it from the object list in the bottom right, but you can't interact with it). Completely quitting Cura and reopening allowed me to select and manipulate the stl (same one as before). I could switch to the monitor tab, back to prepare with no issues. I was able to slice, reslice, change tabs, no problems. The printer was actually turned off during this though (it was also off in the first instance of Cura as well).

I turned my printer ON, switched to the monitor tab, back to prepare, and could not select the model again.
[cura.log](https://github.com/Ultimaker/Cura/files/5598138/cura.log)

**Comment by SmithyAT** [comment]
I see the same issue here, Cura 4.8 and Big Sur 11.0.1.
First I was not able to reproduce it, a user in the forum had the problem, so I looked deeper on it on my system. It has nothing do to with the monitor tab, which I don't use very often. It occurs when I leave Cura open for some time. For example I sliced and printed something on Friday and didn't close Cura. Today (Sunday) I tried to select the part on the build plate I printed on Friday and I was not able to do it anymore. Between Friday and Sunday, I did nothing with Cura.

I think you should be able to reproduce it when you just do nothing for one or two days.

**Comment by nikolafamit** [comment]
> I see the same issue here, Cura 4.8 and Big Sur 11.0.1.
> First I was not able to reproduce it, a user in the forum had the problem, so I looked deeper on it on my system. It has nothing do to with the monitor tab, which I don't use very often. It occurs when I leave Cura open for some time. For example I sliced and printed something on Friday and didn't close Cura. Today (Sunday) I tried to select the part on the build plate I printed on Friday and I was not able to do it anymore. Between Friday and Sunday, I did nothing with Cura.
> 
> I think you should be able to reproduce it when you just do nothing for one or two days.

Yes. The same situation happened to me. Although, it wasn’t for days but in my case Cura was opened for 6-7h

**Comment by fieldOfView** [comment]
> just do nothing for one or two days.

Yeah, that's not going to happen in a hurry ;-)

**Comment by Gatix** [comment]
So I have the same issue, Cura 4.8 and Big Sur 11.0.1. I'm on a Macbook Pro with an external monitor.

If the Cura window is on the external monitor, I can't select object nor click on the axes. Moving the window to the laptop screen fixes the issue. Perhaps the difference in pixel density or resolution is causing the issue?

**Comment by SmithyAT** [comment]
Interesting observation that moving the window to the laptop fixes the issue.
I can reproduce it on my iMac with default resolution but have not seen it yet on my MacBook Pro.

**Comment by evtrados** [comment]
Developers, see internal issue CURA-7897.

**Comment by meichthys** [comment]
I can also confirm that the issue happens on my Macbook Pro (Big Sur) without any external peripherals attached.

**Comment by meichthys** [comment]
I can also reproduce the issue every time using the following steps:

1. Open Cura
2. Add stl to build plate and select model
3. Click the following menu icon:
<img width="56" alt="image" src="https://user-images.githubusercontent.com/10717998/100655757-b6436080-3319-11eb-863e-cd1c44c75fb5.png">
4. Click/Drag on the build plate (Not on a model)
At this point the models bill become un-selectable.

**Comment by SmithyAT** [comment]
Bingo, with these steps I can reproduce it easily.

**Comment by nallath** [comment]
We've also been able to reproduce it (at least) once on our side. I still don't quite get how this is only happening with big Sur though.

**Comment by SmithyAT** [comment]
Thanks Jaime for the update. If there is anything I can do or help, just let me know.

**Comment by Rukongai** [comment]
I get this one a lot. I have to restart the application every 15-20 minutes. Is there any logs I can provide to help track this down?

**Comment by nallath** [comment]
I'm not the product owner, so I don't decide the priority of issues. But I (and our QA engineers as well) have pushed for this to get priority. As it stands I expect that this will be picked up for the 4.9 release. 

 Right now it looks like that we have good/reliable reproduction steps, which is usually enough to fix the issue.

**Comment by GuyJanssen** [comment]
I have exactly the same problem and it popped up after Big Sur was installed and I still had Cura 4.7. It is not solved by moving the window from my external screen back to my MacBook screen.
The problem rises when I add per model settings or even just click on the icon. I use per model settings a lot so I practically have to restart Cura for every object which is pretty often in a 3D printing business. Hopefully you will solve this in 4.8.1 and not in 4.9!

**Comment by yzorgone** [comment]
i have the problem too since Big Sur.

**Comment by timonwehmann** [comment]
I have the same problem as well. Occurred also on Catalina, but not as predictable as it manifests on Big Sur.

**Comment by fabiopdm** [comment]
Using some heavy 3D Models, I have the same issue since Big Sur. Using the last version of Cura, 4.8.0.

When I click to slice or sometimes when I ask to rotate de model to put the face on the plate the Cura software freezes.

**Comment by gzumwalt** [comment]
On my iMac running Big Sur, I can only click on a .STL one time to rotate / translate / etc.  After the first time, I can no longer select the .STL.

Also, the preview mode is a quite screen.

**Comment by xking3** [comment]
Same problem here since Update to Big Sur.

**Comment by nuaimat** [comment]
As a walkaround, I downgraded to 4.7.1, hope this bug will be resolved soon.

**Comment by GuyJanssen** [comment]
On Big Sur version 4.7.1 has the same problems

**Comment by nuaimat** [comment]
@GuyJanssen 4.7.1 worked for me on Big Sur, yes sometimes I have to select objects by clicking on their name in the "Object list" on left-bottom, but I was able to add all needed no-support areas of https://www.thingiverse.com/thing:4239135 and slice it. I couldn't do the same on 4.8

**Comment by GuyJanssen** [comment]
thanks nuaimat, it still is a pain in the ..... really hope they will fix this very soon ........

**Comment by bkarr123** [comment]
I can confirm I am having the same issue. I just updated to Bug Sur 11.1, in Cura 4.8.0 I am having trouble moving  around models, scale, rotate, or anything else. Please help!

**Comment by smaugs** [comment]
same here

**Comment by Salty-Doug** [comment]
Same Issue here with a MacBook Pro mid-2014 and Big Sur Cura 4.8.0.
Here is my log file:
[cura.log.zip](https://github.com/Ultimaker/Cura/files/5760808/cura.log.zip)

**Comment by rhukster** [comment]
Jumping on the dog pile... restarting cura is quickfix but getting annoying!

**Comment by GuyJanssen** [comment]
Hi Ultimaker, is there any sight on a solution??? It is getting pretty annoying running a couple of Ultimaker printers like this!!

**Comment by Thalioss** [comment]
I have the same issue. REALLY annoying to have to quit and restart Cura every time I send a job to a printer.

**Comment by fieldOfView** [comment]
Could you try turning on the `Force layer view compatibility mode (restart required)` option in the General preferences?

**Comment by GuyJanssen** [comment]
> Could you try turning on the `Force layer view compatibility mode (restart required)` option in the General preferences?

I did try that but that doesn't help!

**Comment by fieldOfView** [comment]
> I did try that but that doesn't help!

And - just to be sure - you did restart Cura so the setting had effect?

**Comment by GuyJanssen** [comment]
Yep. After turning it on I restarted and loaded a multi-object 3mf project and could not even select the objects. After turning the flag back off , restarting and loading the same 3mf I could select the separate objects.After clicking the settings per object icon on the left I could not select the objects anymore. So still the same problem!

**Comment by Lukapp** [comment]
Same problem. I can reproduce it without having to restart the application. It is blocked only with the support icon that you comment.

But I have a plugin installed, "Custom Printjob Naming", which allows me to unlock the model without having to restart. This plugin allows us to save some characteristics we are interested in the filename, but it is not necessary to use it, it is only to open a pop-up window. Maybe works with another plugin that opens a popup window.

Unblock
1. "Custom Printjob Naming" plugin installed or "maybe" another plugin that opens a popup window.

2. Select a model with Command-A and select any other button (scale, move), not the problematic icon.

3. Open the plugin properties window by clicking on the name to save. Cancel or OK.

4. Select the model again and it is already unlocked.

https://user-images.githubusercontent.com/8763220/104058236-b590ca80-51f3-11eb-95e5-bcf7a71c4212.mov




 I don't know if it will work with other plugins that open a popup.

Maybe the icon function loses focus or does not return it, and freezes, but with this external plugin or others when you open a window and close, it regain focus of the window, I don't know. But the problem must be in the function of the icon.

![image](https://user-images.githubusercontent.com/8763220/104058374-e8d35980-51f3-11eb-80e5-b665f3fcd7ee.png)

**Comment by GuyJanssen** [comment]
hi @Lukapp, works like a charm!! Thanks so much for your workaround!!  Will save me a whole lot of time!!
@fieldOfView Now Ultimaker hopefully knows what to do about it!

**Comment by Ghostkeeper** [comment]
It's not a solution for us to open up a pop-up every 10 seconds or so to unlock scene interactions in case they have gotten stuck recently.

What we know so far is that apparently some elements are still catching mouse events even though they are invisible, but only on MacOS Big Sur.
We also know that Big Sur is not yet supported in Qt5 either: https://doc.qt.io/qt-5/supported-platforms.html This is probably why it's giving so much problems for us. Qt's own developers are not confident enough with Big Sur bugs that they'd call it "supported", even though they've been working on fixes here and there. Aside from that, Cura is still on a fairly old version of Qt as well: 5.10. This has been necessary so far to be able to support MacOS 10.11 (El Capitan). Since we see El Capitan's usage dwindling we're considering updating Qt to 5.15. That's about everything we can really do about this issue, so that's the plan.

**Comment by uid01089** [comment]
I faced the same issue on Mac OS Big Sur. As workaround, I
1. Open a stl-file, the object is then on the plate
2. Save it as a Cura Project
3. Close Cura and re-open it
4. Load the saved Cura Project

Then I was able to modify the object.

**Comment by Einstein2150** [comment]
This issue is really old. I had this issues before Big Sur too - maybe not so often but after a standby of my macbook every time ...

**Comment by mbourque** [comment]
+1 I have this problem too. Awaiting a fix.

**Comment by lewsid** [comment]
Throwing another hopeful whimper onto the pile, this is a workflow-breaking bug. Happens 100% of the time after saving anything. Constantly closing and re-opening the app is a bummer of a work-around, those delays add up.

**Comment by Ghostkeeper** [comment]
We've just made a testing build with Qt 5.15 and confirmed that this indeed fixes the Big Sur issue. Progress with the upgrade of the Qt version is still ongoing.

**Comment by enegron** [comment]
uid01089 comment describing "save as project" workaround is working for me, however I assume this will be fixed since the rotate feature is available and can be activated after loading an STL file.

**Comment by colomonkey** [comment]
I have the same problem after saving to sd card. The models cannot be selected anymore and the only way to proceed is to restart Cura. Using  Cura 4.8.0 on latest version of Macos Catalina.

**Comment by dJOS1475** [comment]
I have this issue on 4.8 running on MacOS 11.2 also - I'm getting really sick and tired of having to quit Cura and re-run it every time this happens! 👎 

This bug has been present for months, please fix it already!

**Comment by see3d** [comment]
I also have this problem with my Mac Mini running Big Sur.

**Comment by fasihrana** [comment]
Same issue. The preview window is also blank if I slice after opening any dialogue boxes.

**Comment by jamiebikies** [comment]
Same issue for me 🖐️

**Comment by holstvoogd** [comment]
I can reproduce this consistently in 1 easy steps: go to monitor tab (with or without a model loaded even).

Now both the prepare and preview modes are broken.
I use the sidebare GUI pugin & Octroprint.

The logs do contain a bunch of errors related to both:
```
2021-04-03 15:37:48,886 - DEBUG - [MainThread] UM.Controller.setActiveStage [180]: Setting active stage to MonitorStage
2021-04-03 15:37:48,918 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Users/arthur/Library/Application Support/cura/4.8/plugins/SidebarGUIPlugin/SidebarGUIPlugin/resources/qml/MonitorStageMenu.qml:31: TypeError: Cannot read property 'children' of undefined
2021-04-03 15:37:49,510 - DEBUG - [MainThread] cura.AutoSave._onTimeout [59]: Autosaving preferences, instances and profiles
2021-04-03 15:37:52,775 - DEBUG - [MainThread] UM.Controller.setActiveStage [180]: Setting active stage to PrepareStage
2021-04-03 15:37:52,801 - DEBUG - [MainThread] cura.Machines.Models.NozzleModel._update [28]: Updating NozzleModel.
2021-04-03 15:37:52,909 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Users/arthur/Library/Application Support/cura/4.8/plugins/SidebarGUIPlugin/SidebarGUIPlugin/resources/qml/ViewOptionsPanel.qml:148:13: QML QQuickItem: Binding loop detected for property "children"
2021-04-03 15:37:52,913 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Users/arthur/Library/Application Support/cura/4.8/plugins/SidebarGUIPlugin/SidebarGUIPlugin/resources/qml/ViewOptionsPanel.qml:148:13: QML QQuickItem: Binding loop detected for property "children"
2021-04-03 15:37:52,917 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Users/arthur/Library/Application Support/cura/4.8/plugins/SidebarGUIPlugin/SidebarGUIPlugin/resources/qml/ViewOptionsPanel.qml:148:13: QML QQuickItem: Binding loop detected for property "children"
2021-04-03 15:37:52,921 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Users/arthur/Library/Application Support/cura/4.8/plugins/SidebarGUIPlugin/SidebarGUIPlugin/resources/qml/ViewOptionsPanel.qml:148:13: QML QQuickItem: Binding loop detected for property "children"
2021-04-03 15:37:52,934 - DEBUG - [MainThread] cura.CuraApplication.log [1721]: SidebarGUIPlugin patching interface for Cura 4.7 and newer
2021-04-03 15:37:52,944 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Applications/Ultimaker Cura.app/Contents/MacOS/resources/qml/Menus/ConfigurationMenu/CustomConfiguration.qml:234: TypeError: Cannot read property 'activeStack' of undefined
2021-04-03 15:37:53,006 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Users/arthur/Library/Application Support/cura/4.8/plugins/OctoPrintPlugin/OctoPrintPlugin/qml/MonitorItem4x.qml:88: ReferenceError: stop is not defined
2021-04-03 15:37:53,011 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Users/arthur/Library/Application Support/cura/4.8/plugins/OctoPrintPlugin/OctoPrintPlugin/qml/MonitorItem4x.qml:72: TypeError: Cannot read property of null
2021-04-03 15:37:53,015 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Users/arthur/Library/Application Support/cura/4.8/plugins/OctoPrintPlugin/OctoPrintPlugin/qml/MonitorItem4x.qml:156: TypeError: Cannot read property of null
2021-04-03 15:37:53,022 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Users/arthur/Library/Application Support/cura/4.8/plugins/OctoPrintPlugin/OctoPrintPlugin/qml/MonitorItem4x.qml:167: TypeError: Cannot read property of null
2021-04-03 15:37:53,028 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Users/arthur/Library/Application Support/cura/4.8/plugins/OctoPrintPlugin/OctoPrintPlugin/qml/MonitorItem4x.qml:168: TypeError: Cannot read property of null
2021-04-03 15:37:53,034 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Users/arthur/Library/Application Support/cura/4.8/plugins/OctoPrintPlugin/OctoPrintPlugin/qml/MonitorItem4x.qml:204: TypeError: Cannot read property of null
2021-04-03 15:37:53,039 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Users/arthur/Library/Application Support/cura/4.8/plugins/OctoPrintPlugin/OctoPrintPlugin/qml/MonitorItem4x.qml:205: TypeError: Cannot read property of null
2021-04-03 15:37:53,045 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Users/arthur/Library/Application Support/cura/4.8/plugins/OctoPrintPlugin/OctoPrintPlugin/qml/MonitorItem4x.qml:118: TypeError: Cannot read property of null
2021-04-03 15:37:53,050 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Users/arthur/Library/Application Support/cura/4.8/plugins/OctoPrintPlugin/OctoPrintPlugin/qml/MonitorItem4x.qml:185: TypeError: Cannot read property of null
2021-04-03 15:37:53,054 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Users/arthur/Library/Application Support/cura/4.8/plugins/OctoPrintPlugin/OctoPrintPlugin/qml/MonitorItem4x.qml:187: TypeError: Cannot read property of null
2021-04-03 15:37:53,057 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Applications/Ultimaker Cura.app/Contents/MacOS/resources/qml/RoundedRectangle.qml:68: TypeError: Cannot read property of null
2021-04-03 15:37:53,061 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Applications/Ultimaker Cura.app/Contents/MacOS/resources/qml/PrinterOutput/OutputDeviceHeader.qml:36: TypeError: Cannot read property of null
2021-04-03 15:37:53,064 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Applications/Ultimaker Cura.app/Contents/MacOS/resources/qml/PrinterOutput/OutputDeviceHeader.qml:37: TypeError: Cannot read property of null
2021-04-03 15:37:53,069 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Applications/Ultimaker Cura.app/Contents/MacOS/resources/qml/PrinterOutput/OutputDeviceHeader.qml:49: TypeError: Cannot read property of null
2021-04-03 15:37:53,073 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Applications/Ultimaker Cura.app/Contents/MacOS/resources/qml/PrinterOutput/OutputDeviceHeader.qml:59: TypeError: Cannot read property of null
2021-04-03 15:37:53,079 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Applications/Ultimaker Cura.app/Contents/MacOS/resources/qml/PrinterOutput/OutputDeviceHeader.qml:61: TypeError: Cannot read property of null
2021-04-03 15:37:53,083 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Applications/Ultimaker Cura.app/Contents/MacOS/resources/qml/PrinterOutput/OutputDeviceHeader.qml:63: TypeError: Cannot read property of null
2021-04-03 15:37:53,087 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Applications/Ultimaker Cura.app/Contents/MacOS/resources/qml/PrinterOutput/ExtruderBox.qml:20: TypeError: Cannot read property of null
2021-04-03 15:37:53,092 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Applications/Ultimaker Cura.app/Contents/MacOS/resources/qml/PrinterOutput/HeatedBedBox.qml:29: TypeError: Cannot read property of null
2021-04-03 15:37:53,097 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Applications/Ultimaker Cura.app/Contents/MacOS/resources/qml/PrinterOutput/HeatedBedBox.qml:30: TypeError: Cannot read property of null
2021-04-03 15:37:53,103 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Applications/Ultimaker Cura.app/Contents/MacOS/resources/qml/PrinterOutput/HeatedBedBox.qml:40: TypeError: Cannot read property of null
2021-04-03 15:37:53,108 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Applications/Ultimaker Cura.app/Contents/MacOS/resources/qml/PrinterOutput/HeatedBedBox.qml:73: TypeError: Cannot read property of null
2021-04-03 15:37:53,114 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Applications/Ultimaker Cura.app/Contents/MacOS/resources/qml/PrinterOutput/HeatedBedBox.qml:135: TypeError: Cannot read property of null
2021-04-03 15:37:53,118 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Applications/Ultimaker Cura.app/Contents/MacOS/resources/qml/PrinterOutput/HeatedBedBox.qml:246: TypeError: Cannot read property of null
2021-04-03 15:37:53,122 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Applications/Ultimaker Cura.app/Contents/MacOS/resources/qml/PrinterOutput/HeatedBedBox.qml:247: TypeError: Cannot read property of null
2021-04-03 15:37:53,126 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Applications/Ultimaker Cura.app/Contents/MacOS/resources/qml/PrinterOutput/HeatedBedBox.qml:173: TypeError: Cannot read property of null
2021-04-03 15:37:53,130 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Applications/Ultimaker Cura.app/Contents/MacOS/resources/qml/PrinterOutput/HeatedBedBox.qml:175: TypeError: Cannot read property of null
2021-04-03 15:37:53,134 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Applications/Ultimaker Cura.app/Contents/MacOS/resources/qml/PrinterOutput/HeatedBedBox.qml:190: TypeError: Cannot read property of null
2021-04-03 15:37:53,138 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Applications/Ultimaker Cura.app/Contents/MacOS/resources/qml/PrinterOutput/HeatedBedBox.qml:193: TypeError: Cannot read property of null
2021-04-03 15:37:53,144 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Applications/Ultimaker Cura.app/Contents/MacOS/resources/qml/PrinterOutput/ManualPrinterControl.qml:60: TypeError: Cannot read property of null
2021-04-03 15:37:53,149 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Applications/Ultimaker Cura.app/Contents/MacOS/resources/qml/PrinterOutput/ManualPrinterControl.qml:248: TypeError: Cannot read property of null
2021-04-03 15:37:53,152 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Applications/Ultimaker Cura.app/Contents/MacOS/resources/qml/PrinterOutput/ManualPrinterControl.qml:294: TypeError: Cannot read property of null
2021-04-03 15:37:53,156 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Applications/Ultimaker Cura.app/Contents/MacOS/resources/qml/PrinterOutput/MonitorSection.qml:26: TypeError: Cannot read property of null
2021-04-03 15:37:53,159 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Applications/Ultimaker Cura.app/Contents/MacOS/resources/qml/PrinterOutput/MonitorSection.qml:27: TypeError: Cannot read property of null
2021-04-03 15:37:53,164 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Applications/Ultimaker Cura.app/Contents/MacOS/resources/qml/PrinterOutput/ManualPrinterControl.qml:388: TypeError: Cannot read property of null
2021-04-03 15:37:53,168 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Applications/Ultimaker Cura.app/Contents/MacOS/resources/qml/PrinterOutput/ManualPrinterControl.qml:390: TypeError: Cannot read property of null
2021-04-03 15:37:53,172 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Applications/Ultimaker Cura.app/Contents/MacOS/resources/qml/PrinterOutput/ManualPrinterControl.qml:391: TypeError: Cannot read property of null
2021-04-03 15:37:53,178 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Applications/Ultimaker Cura.app/Contents/MacOS/resources/qml/PrinterOutput/MonitorSection.qml:26: TypeError: Cannot read property of null
2021-04-03 15:37:53,183 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Applications/Ultimaker Cura.app/Contents/MacOS/resources/qml/PrinterOutput/MonitorSection.qml:27: TypeError: Cannot read property of null
2021-04-03 15:37:53,186 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Applications/Ultimaker Cura.app/Contents/MacOS/resources/qml/PrinterOutput/MonitorItem.qml:24: TypeError: Cannot read property of null
2021-04-03 15:37:53,190 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Applications/Ultimaker Cura.app/Contents/MacOS/resources/qml/PrinterOutput/MonitorItem.qml:30: TypeError: Cannot read property of null
2021-04-03 15:37:53,195 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Applications/Ultimaker Cura.app/Contents/MacOS/resources/qml/PrinterOutput/MonitorItem.qml:39: TypeError: Cannot read property of null
2021-04-03 15:37:53,200 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Applications/Ultimaker Cura.app/Contents/MacOS/resources/qml/PrinterOutput/MonitorItem.qml:24: TypeError: Cannot read property of null
2021-04-03 15:37:53,206 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Applications/Ultimaker Cura.app/Contents/MacOS/resources/qml/PrinterOutput/MonitorItem.qml:30: TypeError: Cannot read property of null
2021-04-03 15:37:53,214 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Applications/Ultimaker Cura.app/Contents/MacOS/resources/qml/PrinterOutput/MonitorItem.qml:39: TypeError: Cannot read property of null
2021-04-03 15:37:53,221 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Applications/Ultimaker Cura.app/Contents/MacOS/resources/qml/PrinterOutput/MonitorItem.qml:24: TypeError: Cannot read property of null
2021-04-03 15:37:53,225 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Applications/Ultimaker Cura.app/Contents/MacOS/resources/qml/PrinterOutput/MonitorItem.qml:30: TypeError: Cannot read property of null
2021-04-03 15:37:53,230 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Applications/Ultimaker Cura.app/Contents/MacOS/resources/qml/PrinterOutput/MonitorItem.qml:39: TypeError: Cannot read property of null
2021-04-03 15:37:53,235 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Users/arthur/Library/Application Support/cura/4.8/plugins/OctoPrintPlugin/OctoPrintPlugin/qml/MonitorItem4x.qml:215: TypeError: Cannot read property of null
2021-04-03 15:37:53,240 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Applications/Ultimaker Cura.app/Contents/MacOS/resources/qml/RoundedRectangle.qml:68: TypeError: Cannot read property of null
2021-04-03 15:37:53,245 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Applications/Ultimaker Cura.app/Contents/MacOS/resources/qml/MonitorButton.qml:166: TypeError: Cannot read property of null
2021-04-03 15:37:53,250 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Applications/Ultimaker Cura.app/Contents/MacOS/resources/qml/MonitorButton.qml:167: TypeError: Cannot read property of null
2021-04-03 15:37:53,254 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Applications/Ultimaker Cura.app/Contents/MacOS/resources/qml/MonitorButton.qml:178: TypeError: Cannot read property of null
2021-04-03 15:37:53,258 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Applications/Ultimaker Cura.app/Contents/MacOS/resources/qml/MonitorButton.qml:231: TypeError: Cannot read property of null
2021-04-03 15:37:53,262 - WARNING - [MainThread] UM.Qt.QtApplication.__onQmlWarning [406]: file:///Applications/Ultimaker Cura.app/Contents/MacOS/resources/qml/MonitorButton.qml:241: TypeError: Cannot read property of null
2021-04-03 15:38:01,220 - INFO - [MainThread] cura.CuraApplication.readLocalFile [1733]: Attempting to read file file:///Users/arthur/Projects/3D Printers/Voron-2/STLs/TEST_PRINTS/Voron_Design_Cube_v7.stl
```

**Comment by Ghostkeeper** [comment]
The day before yesterday we released a beta of Cura 4.9: https://github.com/Ultimaker/Cura/releases/tag/4.9-beta

In our own findings, this issue is fixed with this beta. Would you like to test as well?
There is still an issue with some drop-downs in that beta, which is specific to MacOS. But it's usable at least. We're trying to fix those up for the stable release.

**Comment by Salty-Doug** [comment]
Yes, I would like to test this beta.

**Comment by GuyJanssen** [comment]
I installed the beta but it keeps crashing when loading an object. I sent the error messages flashing on the screen. So still no solution so far! Very disappointing! Running Big Sur on Mac mini M1.

**Comment by Ghostkeeper** [comment]
The M1 computer is indeed still not supported. We consider this to be a separate issue. There is nothing we can really do about it but wait for Qt to support it (https://bugreports.qt.io/browse/QTBUG-85279 among others).

**Comment by ChrisTerBeke** [comment]
I'm running an M1 MacBook Pro, and everything works fine for me in both Cura 4.8 and 4.9 Beta, with the exception of large parts in the layer preview. I found Rosetta to be very good at emulation actually, haven't found a single app yet that didn't run at all...

**Comment by Salty-Doug** [comment]
Hi,
Where can I download the Cura 4.9 Beta app for Mac?

Fair Winds and Following Seas

Salty Doug

**Comment by Rukongai** [comment]
@Salty-Doug 

It's under releases on this repo.

https://github.com/Ultimaker/Cura/releases

**Comment by Salty-Doug** [comment]
I found it. 
Thanks.

**Comment by Ghostkeeper** [comment]
Aside from a response that https://github.com/Ultimaker/Cura/issues/9542 is indeed a bug, we didn't get any response in this ticket whether 4.9 fixes this issue. But our own tests point out that the issue is resolved by the upgrade of Qt to 5.15.

**Comment by Salty-Doug** [comment]
Thanks to the Dev Team. The problem is fixed on my Mac running Big Sur.
YEAH!!!

**Comment by dJOS1475** [comment]
Yes, it is fixed, however, 4.9 crashes frequently, even when not in use.

**Comment by jlkorpal** [comment]
I do not know if this issue has been resolved. I have had this issues on/off for awhile. The way I was able to get Cura to work on my MAC is to delete the files under ~/Library/Application Support/cura. once the files have been delete Cura functions normally. But over time the issue returns. It seems the issue is a memory leak and over time it corrupts one or more files leaving Cura useless.

**Comment by fieldOfView** [comment]
@jlkorpal I think you can fix it more permanently by unchecking the "Restore window position on start" checkbox in the General preferences. This makes sure the application always opens on your primary display (assuming you have multiple).

I think the "over time the issue returns" is caused by you over time moving Cura to a secondary display before closing it. On subsequent launched, Cura would open on that display which may be the primary cause of your issue. Removing the whole configuration would also reset the window position on launch, "fixing" Cura until you run into the issue again that is caused by Cura starting on the secondary display.
